projects/packages/post-list/src/class-post-list.php

```diff
@@ -0,0 +1,119 @@
+<?php
+/**
+ * The Post List Admin Area.
+ *
+ * @package automattic/jetpack-post-list
+ */
+
+namespace Automattic\Jetpack\Post_List;
+
+use Automattic\Jetpack\Assets;
+
+/**
+ * The Post_List Admin Area
+ */
+class Post_List {
+	/**
+	 * The configuration method that is called from the jetpack-config package.
+	 */
+	public static function configure() {
+		$post_list = self::get_instance();
+		$post_list->register();
+	}
+
+	/**
+	 * Initialize the Post List UI.
+	 *
+	 * @return Post_List Post_List instance.
+	 */
+	public static function get_instance() {
+		return new Post_List();
+	}
+
+	/**
+	 * Sets up Post List action callbacks if needed.
+	 */
+	public function register() {
+		if ( ! did_action( 'jetpack_on_posts_list_init' ) ) {
+			add_action( 'admin_enqueue_scripts', array( $this, 'enqueue_scripts' ) );
+			add_action( 'in_admin_footer', array( $this, 'create_app_root_element' ) );
+
+			/**
+			 * Action called after initializing Post_List Admin resources.
+			 *
+			 * @since $$next-version$$
+			 */
+			do_action( 'jetpack_on_posts_list_init' );
+		}
+	}
+
+	/**
+	 * Enqueue scripts depending on the post-list query var.
+	 *
+	 * @param string $hook Page hook.
+	 */
+	public function enqueue_scripts( $hook ) {
+		if ( 'edit.php' === $hook ) {
+			$plugin_path = Assets::get_file_url_for_environment( './index.js', './index.js', __FILE__ );
+
+			wp_enqueue_script(
+				'jetpack_posts_list_ui_script',
+				$plugin_path,
+				array(),
+				gmdate( 'YW' ),
```

Comment:
Instead of having an ever-changing version, what would you say about using the package's version?

You can check the JITM package for an example of how that version is set, used, and kept up to date with each new release:
https://github.com/Automattic/jetpack/blob/a1c6fd7aa74f050172ea0d1721b41f9272af0ab4/projects/packages/jitm/composer.json#L64
https://github.com/Automattic/jetpack/blob/f7bc00e2266e322d20bdf1e48c3014bf02e80445/projects/packages/jitm/src/class-jitm.php#L23
https://github.com/Automattic/jetpack/blob/f7bc00e2266e322d20bdf1e48c3014bf02e80445/projects/packages/jitm/src/class-jitm.php#L156

Reply:
Sounds good to me! I implemented it similar to the JITM package. 👍